make_wheels.py

```diff
@@ -134,20 +134,33 @@ def write_ziglang_wheel(out_dir, *, version, platform, archive):
         name='ziglang',
         version=version,
         tag=f'py3-none-{platform}',
-        metadata={
-            'Summary': 'Zig is a general-purpose programming language and toolchain for maintaining robust, optimal, and reusable software.',
-            'Description-Content-Type': 'text/markdown',
-            'License': 'MIT',
-            'Classifier': [
-                'License :: OSI Approved :: MIT License',
-            ],
-            'Project-URL': [
-                'Homepage, https://ziglang.org',
-                'Source Code, https://github.com/ziglang/zig-pypi',
-                'Bug Tracker, https://github.com/ziglang/zig-pypi/issues',
-            ],
-            'Requires-Python': '~=3.5',
-        },
+        metadata=[
+            ('Summary', 'Zig is a general-purpose programming language and toolchain for maintaining robust, optimal, and reusable software.'),
+            ('Description-Content-Type', "'text/markdown'; charset=UTF-8; variant=GFM"),
+            ('License-Expression', 'MIT'),
+            ('License-File', 'LICENSE'),
+            ('License-File', 'ziglang/lib/libc/glibc/LICENSES'),
+            ('License-File', 'ziglang/lib/libc/mingw/COPYING'),
+            ('License-File', 'ziglang/lib/libc/musl/COPYRIGHT'),
+            ('License-File', 'ziglang/lib/libc/wasi/LICENSE'),
+            ('License-File', 'ziglang/lib/libc/wasi/LICENSE-APACHE'),
+            ('License-File', 'ziglang/lib/libc/wasi/LICENSE-APACHE-LLVM'),
+            ('License-File', 'ziglang/lib/libc/wasi/LICENSE-MIT'),
+            ('License-File', 'ziglang/lib/libcxx/LICENSE.TXT'),
+            ('License-File', 'ziglang/lib/libcxxabi/LICENSE.TXT'),
+            ('License-File', 'ziglang/lib/libunwind/LICENSE.TXT'),
+            ('Classifier', 'Development Status :: 4 - Beta'),
+            ('Classifier', 'Intended Audience :: Developers'),
+            ('Classifier', 'Topic :: Software Development :: Compilers'),
+            ('Classifier', 'Topic :: Software Development :: Code Generators'),
+            ('Classifier', 'Topic :: Software Development :: Build Tools'),
+            ('Classifier', 'Programming Language :: Other'),
+            ('Classifier', 'Programming Language :: Other Scripting Engines'),
```

Comment:
These are some new classifiers. I can split this into another PR or remove them if not needed. I do think they fit with the greater goal of the project, however; distributions on PyPI with such classifiers have compilers and so on and the like.

Reply:
Additionally, it might make sense to add Python version classifiers here.

Reply:
> Additionally, it might make sense to add Python version classifiers here.

The package is intended to work on "any" Python version so I disagree. A generic "Python 3" classifier could work, but it's probably not important.

Reply:
Makes sense – I had thought about it from how it would be a maintenance burden, too, but we can leave this. 👍